homework/index.js

```diff
@@ -32,39 +35,59 @@
 
   const root = document.getElementById('root');
   const header = createAndAppend('header', root, { class: 'header' });
-  createAndAppend('p', header, { text: 'HYF Repositories' });
 
-  const select = createAndAppend('select', header, { id: 'selector' });
-  const option = x => {
-    for (let i = 0; i < x.length; i++) {
-      createAndAppend('option', select, { text: x[i].name, value: i });
+  function createHeader() {
+    createAndAppend('p', header, { text: 'HYF Repositories' });
+  }
+  createHeader();
+
+  function createOptions(repositoryList) {
+    const select = createAndAppend('select', header, { id: 'selector' });
+    for (let i = 0; i < repositoryList.length; i++) {
+      const repository = repositoryList[i];
+      createAndAppend('option', select, { text: repository.name, value: i });
     }
-  };
+  }
 
   const mainContainer = createAndAppend('div', root, { id: 'container' });
-  const leftContainer = createAndAppend('div', mainContainer, { class: 'left-block frame' });
-  const table = createAndAppend('table', leftContainer);
-  const listContributor = createAndAppend('tbody', table);
+  function createListContributor() {
+    const leftContainer = createAndAppend('div', mainContainer, { class: 'left-block frame' });
+    const table = createAndAppend('table', leftContainer);
+    const listContributor = createAndAppend('tbody', table);
```

Comment:
It looks like you are returning a reference to a `tbody` element; the name `listContributor` suggest that.

Reply:
the table will contain the list of contributors.